UEBPrep/app/controllers/cards_controller.rb

```diff
@@ -21,16 +21,16 @@ def new
 
   # GET /cards/1/edit
   def edit
+    @card = Card.find(params[:id])
   end
 
   # POST /cards
   def create
-    #binding.pry
-    case card_params[:cardtype]
+    case create_params[:cardtype]
       when 'text'
-        @content = TextContent.new(:text => card_params[:cardtext], :created_by => current_user.id)
+        @content = TextContent.new(:text => create_params[:cardtext], :created_by => current_user.id)
```

Comment:
refactor to a separate set of params for the create path versus other controller methods.